### PR TITLE
Jetpack Connect: add WP logo in the Jetpack remote install credentials form.

### PR DESCRIPTION
**Pull request by AnnaMag**
Main change introduced here is the WP logo added to `/public/images/wordpress` and included in the credentials form in the remote install flow.

Styling extras:
- use Card component to introduce extra spacing between the form and the footer,
- center the header text.

**To test:**
- checkout this branch on your local Calypso,
- go to `calypso.localhost:3000/jetpack/connect` and verify the visual changes.

**Note:** PLEASE IGNORE password hide icon inside the input field and button color. It is changed in a follow-up PR.

screen shot 2018-02-28 at 22 33 30

### CODE REVIEW

**Comment by drw158** [comment]
Not sure about using the .org logo here. It makes sense, but it seems out of place because we mainly use the simplified logo in Calypso.

Maybe we should check other login forms and be as consistent as possible. Note in the main login form we don't have the icons, and the label is "Email Address or Username".

screen shot 2018-02-28 at 4 49 06 pm

**Comment by drw158** [comment]
Maybe instead of the .org logo, we could pull in their site avatar and site name/url.

**Comment by drw158** [comment]
See more design discussion here: p7rd6c-1dS

**Comment by rickybanister** [comment]
@AnnaMag can we pull their site title and blavatar at this point? We have no connection to the site at this stage.

**Comment by AnnaMag** [comment]
@rickybanister, at this point all we know is the url of a site that has been passed through the `jetpack/connect` screen. Users do not need to have a .com account at this stage.

**Comment by beaulebens** [comment]
> can we pull their site title and blavatar at this point? We have no connection to the site at this stage.

If we wanted to be fancy, we could technically attempt to scrape this info directly from their site. Grab the HTML, pull the `<title>` tag, look for favicons/app icons, etc.

**Comment by AnnaMag** [comment]
At this stage the validity of the credentials is not confirmed, so we cannot be sure that the person who input the url manages the site (or is site's user). Would it be relevant info to display generally?

**Comment by beaulebens** [comment]
I don't think showing the site title/icon will make or break the flow, so I would probably skip it for now. It's nice context to add, but the person presumably just entered the URL, so they should know where they are/what's going on.

File: client/jetpack-connect/remote-credentials.js [hunk 1]
```diff
@@ -89,7 +89,7 @@ export class OrgCredentialsForm extends Component {
 		const { translate } = this.props;
 
 		return (
-			<span className="jetpack-connect__install-step">
+			<span className="jetpack-connect__install-step creds-form">
```
Comment (seear) [inline client/jetpack-connect/remote-credentials.js]: The `creds-form` class here [needs to be prefixed](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/css.md) with `jetpack-connect__`.

**Comment by AnnaMag** [comment]
Thanks for reviews and discussion 💯 

It was decided that the logo will be removed for this iteration. I update the patch with relevant changes. I'll still still merge it as on top of logo it adds needed styling to the credentials form. Also, leaving the added WP logo for future use.

**Pull request merged**